components/CookieConsentBanner.tsx

```
@@ -0,0 +1,320 @@
+                    {t('title')}
+                  </h2>
+                  <p
+                    id="cookie-banner-description"
+                    className="text-sm text-gray-600 dark:text-gray-300 leading-relaxed"
+                  >
+                    {t('description')}{' '}
+                    <button
+                      onClick={() => setShowDetails(true)}
+                      className="text-primary-600 dark:text-primary-400 hover:underline font-medium"
+                      aria-label={t('customizeSettings')}
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)